### PR TITLE
Document passkey "upgrade" behavior

### PR DESCRIPTION
**Pull request by codingllama**
Document why MFA authenticators are sometimes "upgraded" to passkeys, in the hopes that this may answer future questions about the (slightly surprising) behavior.

Related to PRs #45222 and #45252.

#39521

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/admin-guides/access-controls/guides/passwordless.mdx
Comment (ptgott) [inline docs/pages/admin-guides/access-controls/guides/passwordless.mdx:336]:
```suggestion
### Why did my multi-factor authentication (MFA) device become a passkey?
```
Grammar/capitalization

**Review by ptgott — APPROVED** [review]

**Comment by codingllama** [comment]
Thanks everyone. I'll hold this until the features actually land.

**Pull request merged**
